src/components/SignInButtons/GoogleSignIn/index.website.tsx

```diff
@@ -1,28 +1,24 @@
-import PropTypes from 'prop-types';
 import React, {useCallback} from 'react';
 import {View} from 'react-native';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import withLocalize, {WithLocalizeProps} from '@components/withLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as Session from '@userActions/Session';
 import CONFIG from '@src/CONFIG';
 import CONST from '@src/CONST';
 
-const propTypes = {
-    /** Whether we're rendering in the Desktop Flow, if so show a different button. */
-    isDesktopFlow: PropTypes.bool,
-
-    ...withLocalizePropTypes,
-};
-
-const defaultProps = {
-    isDesktopFlow: false,
+type GoogleSignInProps = WithLocalizeProps & {
+    isDesktopFlow: boolean;
```

Comment:
```suggestion
    isDesktopFlow?: boolean;
```

According to propTypes.